src/components/SubNav/SubNav.tsx

```diff
@@ -0,0 +1,312 @@
+  "section.whats-on.primary-05",
+  "dark.brand.primary-05",
+  "dark.section.blogs.primary-05",
+  "dark.section.books-and-more.primary-05",
+  "dark.section.connect.primary-05",
+  "dark.section.education.primary-05",
+  "dark.section.locations.primary-05",
+  "dark.section.research.secondary-05",
+  "dark.section.research-library.lpa-05",
+  "dark.section.research-library.schomburg-05",
+  "dark.section.research-library.schwarzman-05",
+  "dark.section.whats-on.primary-05",
+] as const;
+
+export type actionBackgroundColors = typeof actionBackgroundColorsArray[number];
+
+export const highlightColorsArray = [
+  "brand.primary",
+  "section.blogs.primary",
+  "section.books-and-more.primary",
+  "section.connect.primary",
+  "section.education.primary",
+  "section.locations.primary",
+  "section.research.primary",
+  "section.research-library-lpa.primary",
+  "section.research-library-schomburg.primary",
+  "section.research-library-schwarzman.primary",
+  "section.whats-on.primary",
+  "dark.brand.primary",
+  "dark.section.blogs.primary",
+  "dark.section.books-and-more.primary",
+  "dark.section.connect.primary",
+  "dark.section.education.primary",
+  "dark.section.locations.primary",
+  "dark.section.research.secondary",
+  "dark.section.research-library-lpa.primary",
+  "dark.section.research-library-schomburg.primary",
+  "dark.section.research-library-schwarzman.primary",
+  "dark.section.whats-on.primary",
+];
+export type highlightColors = typeof highlightColorsArray[number];
+
+export interface SubNavProps {
+  /**
+   * The background color to be applied to the hover and active states
+   * of the SubNavLink and SubNavButton components.
+   * This allows for customization of the action items.
+   */
+  actionBackgroundColor?: actionBackgroundColors;
+  /** Additional class name for the `SubNav` component. */
+  className?: string;
+  /**
+   * Custom color for SubNavLink, SubNavButton, and icons.
+   */
+  highlightColor?: highlightColors;
+  /**
+   * Optional unique ID for accessibility, allowing other components
+   * to reference this element.
+   */
+  id?: string;
+  /**
+   * Primary actions displayed on the left side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  primaryActions: React.ReactNode;
+  /**
+   * Secondary actions displayed on the right side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  secondaryActions?: React.ReactNode;
+}
+
+export const SubNavButton: React.FC<React.PropsWithChildren<any>> = ({
+  id,
+  children,
+  isOutlined,
+  isSelected,
+  screenreaderOnlyText = "",
+}) => {
+  const childrenStyles = useMultiStyleConfig("SubNavChildren", {
+    isOutlined: isOutlined,
+  });
+
+  return (
+    <li>
+      <Button
+        id={id}
+        buttonType="text"
+        className={isSelected ? "selectedItem" : ""}
+        sx={{ ...childrenStyles.outLine }}
```

Comment:
You should be able to just have `sx={childrenStyles.outLine}`. It's the same as `{ ...childrenStyles.outLine }`